packages/charting/src/components/StackedBarChart/StackedBarChart.types.ts

```diff
@@ -0,0 +1,105 @@
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IStackedBarChart {}
+
+export interface IStackedBarChartProps {
+  /**
+   * Data to render in the chart.
+   */
+  data?: number[];
```

Comment:
It seems like it would be useful for users to be able to label the sections of the chart rather than only showing the numerical data (40, 25, 53, 87 in the example chart).